xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.P6257-I211.xml

```diff
@@ -0,0 +1,128 @@
+               <xi:fallback>
+                  <p>The Publications Statement should appear here. (c) Brown University</p>
+               </xi:fallback>
+            </xi:include>
+            <idno type="USEpigraphy" xml:id="NJ.Princ.PU.AM.G.P6257-I211">NJ.Princ.PU.AM.G.P6257-I211</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc>
+               <msIdentifier>
+                  <region>NJ</region>
+                  <settlement>Princ</settlement>
+                  <institution>PU</institution>
+                  <repository>AM</repository>
+                  <idno>P6257-I211</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="grc" otherLangs=""/>
+                  <msItem>
+                     <p/>
+                  </msItem>
+               </msContents>
+               
+               <physDesc>
+                  <objectDesc ana="#fragment">
+                     <supportDesc ana="#stone">
+                        <support>
+                           <p> limestone</p>
+                           <dimensions type="surface" unit="cm">
+                              <height>1.0 cm</height>
+                              <width>1.2 cm</width>
+                              <depth>0.9 cm</depth>
+                           </dimensions>
+                        </support>
+                        <condition ana="#fragment.single">
+                           <p/>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="2">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#impressed.inscribed"> </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p/>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               <history>
+                  <summary>
+                     <rs/>  <!-- content can be "fake" or blank -->
+                  </summary>
+                  
+                  <origin>
+                     <date notBefore="0001" notAfter="0200">first-second century CE</date>
+                     <placeName ref="asia.asia_minor">Turkey</placeName>
+                  </origin>
+                  <acquisition>
```

Comment:
We've actually changed how we handle this; for now, best to avoid using the acquisition button in Author mode and I'll writ up some directions on how best to handle this going forward.